WMCDuplicateRemover.Tests/EPG/EPGIntegrationTests.cs

```diff
@@ -1,10 +1,6 @@
-﻿using NUnit.Framework;
+﻿using Moq;
```

Comment:
No, you can see that it's a nuget package if you check the packages.config. Moq is an open source project for creating easy mock objects [Moq's GitHub](https://github.com/moq/moq4)